lineman/app/components/navbar/navbar_search.coffee

```diff
@@ -21,6 +21,9 @@ angular.module('loomioApp').directive 'navbarSearch', ->
       return if $element[0].contains document.activeElement
       $scope.closeSearchDropdown()
 
+    $scope.showDropdown = ->
```

Comment:
Only show the groups dropdown bit if the user is a member of a group, otherwise it shows a weird disembodied floaty thing when you first click on search.